apps/playwright/tests/admin/auth.spec.ts

```diff
@@ -185,6 +240,9 @@ test.describe("Authenticated", () => {
           const page = await context.newPage();
           await page.goto(allowedPath);
           await page.waitForURL(allowedPath);
+          await expect(
+            page.getByRole("link", { name: "Dashboard" }),
+          ).toBeVisible();
```

Comment:
Added in some places to improve test's usefulness. 
I noticed without this, sometimes assertions were succeeding when they shouldn't as it would assert something was not present while the page was still loading.